src/LPM.py

```diff
@@ -297,19 +300,10 @@ def cprint(text, color):
                             print('Loupe folder not found, creating it...')
                             # Add the Loupe package back in there.
                             loupePkg = librariesPkg.addEmptyPackage('Loupe') 
-                        for package in args.packages:
-                            # Extract version info if included (this would show up after the '@' character, i.e. 'mylib@3.0.4')
-                            # TODO: this check is now redundant with a version check done at the top of this file, so this should get refactored...
-                            if package.find('@') > -1:
-                                splitPackage = package.split('@')
-                                packageName = splitPackage[0]
-                                packageVersion = splitPackage[1]
-                                installSource(packageName, packageVersion, loupePkg)                               
-                            else:
-                                packageName = package
-                                installSource(packageName, '', loupePkg)
+                        for (packageFullName, packageBaseName, version) in zip(packagesFullNames, packagesBaseNames, packagesVersions):
```

Comment:
Here you zip them together to iterate over three lists but if you would have kept it in a single list it would be as easy as:

```python
for package in packages:
    installSource(package.fullName, package.version, loupePkg)
```

or with further refactoring to `installSource` function even simpler

```python
for package in packages:
    installSource(package)
```